WEB_SERVICE/WEB/_SECURE/SECURE_WEB_COOKIE_FLAGS/description.md

```diff
@@ -0,0 +1,9 @@
+**Secure Cookie Implementation**: A cookie is considered secure when it has the appropriate security attributes, such as `Secure` and `HttpOnly`. 
+
+- The `Secure` flag ensures that the cookie is only sent over HTTPS, protecting it from being transmitted over unencrypted channels.
+- The `HttpOnly` flag mitigates the risk of Cross-Site Scripting (XSS) attacks by making the cookie inaccessible to JavaScript.
+
+Here’s an example of a secure cookie configuration:
+
```

Comment:
@3asm  Not sure but we can also mention SameSite attribute

Reply:
Yep `SameSite` and `Expiration/Max-Age`